Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/config/create_authinfo.go

```diff
@@ -59,13 +59,13 @@ Specifying a name that already exists will merge new fields on top of existing v
 
 const create_authinfo_example = `// Set only the "client-key" field on the "cluster-admin"
 // entry, without touching other values:
-$ kubectl set-credentials cluster-admin --client-key=~/.kube/admin.key
+$ kubectl config set-credentials cluster-admin --client-key=~/.kube/admin.key
```

Comment:
probably want to start using `%[1]s` in help upstream and passing in the command name like we do in origin